src/Clock.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+
+  type Props = {
+    name: string,
+  };
+
+  type State = {
+    currentTime: Date,
+  };
+
+export class Clock extends React.Component<Props, State> {
+  state: State = {
+    currentTime: new Date(),
+  };
+
+  timerId = 0;
+
+  componentDidMount() {
+    this.timerId = window.setInterval(() => {
+      this.setState({
+        currentTime: new Date(),
+      });
+      // eslint-disable-next-line no-console
+      console.info(this.state.currentTime.toUTCString().slice(-12, -4));
+    }, 1000);
+  }
+
+  componentDidUpdate(prevProps: Props) {
+    if (this.props.name !== prevProps.name) {
+      // eslint-disable-next-line no-console
+      console.debug(`Renamed from ${prevProps.name} to ${this.props.name}`);
+    }
+  }
+
+  componentWillUnmount() {
+    window.clearInterval(this.timerId);
+  }
+
+  render() {
+    return (
+      <div className="Clock">
+        <strong className="Clock__name">
+          {this.props.name}
+        </strong>
+
+        {' time is: '}
+        <div className="Clock__time">
+          {this.state.currentTime.toUTCString().slice(-12, -4)}
+        </div>
```

Comment:
replace by `span` to show in one line
```suggestion
        <spanclassName="Clock__time">
          {this.state.currentTime.toUTCString().slice(-12, -4)}
        </span>
```